src/components/meta.js

```diff
@@ -20,27 +28,33 @@ function Meta({ note, repliesCount, repostsCount, likesCount }) {
       day: 'numeric',
     });
     encodedNoteId = getNoteId(note.id);
+    formattedZapAmount = formatZapAmount(zapAmount);
   }
 
   return (
     <div class="cardMeta">
       <div class="cardDate">{formattedDate}</div>
       <hr />
       <div class="cardInteractions">
-        <div class="interactionContainer">
+        <div class="interactionContainer" title="Amount of Zaps">
```

Comment:
I'd change the title here to "Total sats zapped"

Reply:
Agree